dev-docs/RFCs/v4.0.0/terms_for_concepts_and_datatypes.md

```diff
@@ -31,8 +33,28 @@ Reaching a consensus on the precise, technical language used when discussing H3
 
 ### Notes
 
-We concede that "hexagon" will still be used informally to refer to the concept of "cell" (As the core team, we do it ourselves!). This should be expected in casual, informal discussions of H3. When **precision** is required, however, we advise the use of strict technical terms like "index", "cell", "hexagon", "pentagon", etc.
+We realize that "hexagon" will still be used informally to refer to the concept of "cell" (As the development team, we do it ourselves!). This should be expected in casual, informal discussions of H3. When **precision** is required, however, we advise the use of strict technical terms like "index", "cell", "hexagon", "pentagon", etc.
 
 ## Use of "Topological"
 
-todo: define "topological"
+We should clarify in the documentation that H3 works with **topological** hexagons and pentagons, rather than true **geometric** hexagons and pentagons.
```

Comment:
Can we also clarify here that the vast majority of hexagons (and half the pentagons) are both topological and geometric hexagons (or pentagons)? It's only on icosahedron boundaries that extra points are added to deal with the two gnomonic projections adjacent to each other.

Reply:
adding!